From 8bb15dd46e28a4cc603215c09344cb5a936dbe6f Mon Sep 17 00:00:00 2001
From: Jeevaka Prabu Badrappan <jeevaka.badrappan@intel.com>
Date: Sat, 5 Dec 2020 20:39:46 +0530
Subject: [PATCH 05/40] Revert "ANDROID: kbuild: don't merge
 .*..compoundliteral in modules"

This reverts commit b2e761c8d9dee0cfdd14a72ce797917198dfda97.
---
 scripts/module-lto.lds | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/scripts/module-lto.lds b/scripts/module-lto.lds
index 38cd6cf87ef1..060e067e3ce0 100644
--- a/scripts/module-lto.lds
+++ b/scripts/module-lto.lds
@@ -15,17 +15,17 @@ SECTIONS {
 
 	.bss : {
 		*(.bss .bss.[0-9a-zA-Z_]*)
-		*(.bss..L*)
+		*(.bss..L* .bss..compoundliteral*)
 	}
 
 	.data : {
 		*(.data .data.[0-9a-zA-Z_]*)
-		*(.data..L*)
+		*(.data..L* .data..compoundliteral*)
 	}
 
 	.rodata : {
 		*(.rodata .rodata.[0-9a-zA-Z_]*)
-		*(.rodata..L*)
+		*(.rodata..L* .rodata..compoundliteral*)
 	}
 
 	/*
-- 
2.17.1

